packages/nds-checkbox/nds-checkbox.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module "@nice-digital/nds-checkbox" {
+	import React = require("react");
+
+	export interface Props {
```

Comment:
I think this is better being called CheckboxProps so we can do import { CheckBoxProps } from "@nice-digital/nds-checkbox"; which is less ambiguous